static/app/views/dashboards/widgets/barChartWidget/barChartWidgetSeries.tsx

```diff
@@ -0,0 +1,34 @@
+import BarSeries from 'sentry/components/charts/series/barSeries';
+
+import type {TimeseriesData} from '../common/types';
+
+export function BarChartWidgetSeries(timeserie: TimeseriesData, complete?: boolean) {
+  return complete
```

Comment:
I'm not entirely sure what `complete` is meant to do here and how would I know when to use one or the other? Maybe it's something we can document in a comment